loom/script/serialize/lsBinReader.cpp

```diff
@@ -784,6 +782,9 @@ Assembly *BinReader::loadExecutable(LSLuaState *_vm, utByteArray *byteArray)
     {
         Reference *ref = lmNew(NULL) Reference;
         ref->name     = readPoolString();
+
+        lmAssert(ref->name[0] != 0, "Reference name is empty, try recompiling the executable");
```

Comment:
OK - so it should say something like "invalid .loom file" or "bad script assembly" or whatever?